opentelemetry-sdk/src/opentelemetry/sdk/_metrics/view.py

```diff
@@ -13,8 +13,156 @@
 # limitations under the License.
 
 
-# TODO: #2247
-# pylint: disable=no-self-use
+from fnmatch import fnmatch
```

Comment:
Is this still needed since we decided to throw for wildcard?

Reply:
Yes, a view can be created without a name and with a wildcard in `instrument_name`.